.travis.yml

```diff
@@ -94,6 +94,12 @@ script:
    travis_wait 30 valgrind --error-exitcode=42 ./tests 16 &&
    travis_wait 30 valgrind --error-exitcode=42 ./exhaustive_tests;
    fi
+ - if [ -n "$BENCH" ]; then
```

Comment:
Better to unset `BENCH` int the `BUILD=distcheck` case similar to `CTIMETEST` because the bench binaries are not build in that case.

Reply:
Hmm so why didn't the tests fail

Reply:
Maybe because there's `;` between the ifs? hmmm

Reply:
Got it to fail: https://travis-ci.org/github/elichai/secp256k1/jobs/663963345
Will fix now.
Thanks.

Reply:
ugh yep, I think we want `&&` to make sure that the final exit code of the statement.

I think it's cleaner to have individual lines in yaml even.

Reply:
> I think it's cleaner to have individual lines in yaml even.
Then the execution continues. With `&&` we stop after the first failing command, which is okay but not optimal.

Reply:
Can you split an `if` across different yaml lines?

Reply:
Now, but you could different `if`s with combined conditions on  different lines. If you want to try, feel free to add a commit on top. Otherwise tell us and let's get his merged, it's certainly good as it is.

Reply:
I'm kinda tired of fighting with travis, so unless there's any objections I prefer to get it in as is